suseconnect-ng.spec

```diff
@@ -66,6 +67,21 @@ replaced SUSEConnect.
 %{go_nostrip}
 %{go_provides}
 
+%package -n libsuseconnect
+Summary:        C interface to suseconnect-ng.
+Group:          System/Management
```

Comment:
As you pointed out elsewhere, this is currently missing the requires of the main package as both the cli and lib will call those external tools. We can use a macro to not repeat ourselves or just duplicate them.

Reply:
I didn't find an easy way to do it with macros inside the spec file. We could do some prjconf macros but I'd rather keep this repo self contained.
I added `Requires: suseconnect-ng` to `libsuseconnect` to avoid redundancy.
Bonus side effect is that we won't end up with a system without `SUSEConnect` CLI if we replace old `SUSEConnect` package.